tests/integration/layers/layersManager.spec.ts

```diff
@@ -38,4 +44,196 @@ describe('layerManager', function () {
       expect(response.body).toEqual({ message: notFoundErrorMessage });
     });
   });
+
+  describe('#addLayer', function () {
+    it('Happy Path', async function () {
+      const response = await requestSender.addLayer(mockLayerNameIsNotExists);
+
+      expect(response.status).toBe(httpStatusCodes.CREATED);
+    });
+
+    it('Bad Path', async function () {
+      const mockBadRequestRequest = ({
+        mockName: 'NameIsNotExists',
```

Comment:
add comment of invalid input parameter name

Reply:
fixed in 9b9d912